src/mesido/workflows/io/write_output.py

```diff
@@ -412,7 +442,7 @@ def _name_to_asset(name):
 
         kpis_top_level.kpi.append(
             esdl.DistributionKPI(
-                name="High level cost breakdown [EUR]",
+                name="High level cost breakdown [EUR] (year 1)",
```

Comment:
Generally the capex costs are not made all in the first year, so it feels a bit strange to name it "year 1".
Two options I can think of are; 
1. to only name the yearly opex.
2. to divide the capex by the number of years for the optimisation and then state it as capex averaged over time.

Reply:
Agree "year 1" is weird. I changed the kpis to "yearly averaged" and "optim time horizon year period" for CAPEX and OPEX values